src/components/molecules/Alert/Alert.tsx

```diff
@@ -12,7 +12,9 @@ export interface AlertProps {
   size?: IconSize
   style?: React.CSSProperties
   title: string
+  timerCount?: number
   type: AlertType
+  withTimeout?: boolean
```

Comment:
I guess it's a more common behaviour that the Alert disappear automatically. So maybe it should be the default behaviour and you just provide a `timerCount` and a `stay`(if `true` the `Alert` doesn't disappear) props.

Reply:
okay will do the adjustments